src/app/items/item-tabs.ts

```diff
@@ -79,8 +78,7 @@ export class ItemTabs implements OnDestroy {
         filteredTaskTabs.length === 0 && this.isCurrentTab(childrenEditTab) ? childrenEditTab : null,
         ...taskTabs.map(t => ({ title: t.name, routerLink: [], tag: t.view })),
         this.isCurrentTab(editTab) || (editTabEnabled && hasEditionPerm) ? editTab : null,
-        this.isCurrentTab(groupStatsTab) || (canViewGroupStats && !isTask) ? groupStatsTab : null,
-        this.isCurrentTab(userStatsTab) || (canViewUserStats && !isTask) ? userStatsTab : null,
+        this.isCurrentTab(userOrGroupStatsTab) || (canViewGroupStats || canViewUserStats) && !isTask ? userOrGroupStatsTab : null,
```

Comment:
so you do not need to separate `canViewGroupStats` and `canViewUserStats` I assume. So you can delete these var and have a 
```
const canViewStats = watchedGroup ? canWatchResults : canViewContent);
const showProgress = (!isTask || !disablePlatformProgressOnTasks) && canViewStats;
```